content/departments/product-engineering/engineering/cloud/devops/managed/v1.1/mi1-1_restore_process.md

```diff
@@ -0,0 +1,84 @@
+# Restore a managed instance v1.1 when it is completely gone
+
+<span class="badge badge-note">SOC2/CI-110</span>
+
+> Are you experiencing a zonal failure? Follow the [failover process](./mi1-1_failover_process.md)
+
+In MI v1.1, we now have resources running outside the VM, i.e Cloud SQL, hence the restoration comes with multiple stages. Perform the restoration in the following order.
+
+## Restoring Cloud SQL
+
+Use cases:
+
+- Cloud SQL data is corrupted by a broken database migration
+- Cloud SQL data is deleted
```

Comment:
Do we skip the full deletion of CloudSQL instance? I think we cannot destroy it via terraform b/c of `delete lock protection`, so imho we should be safe.